Sources/Conduit/Networking/URLSessionClient.swift

```diff
@@ -17,6 +17,7 @@ private typealias SessionCompletionHandler = (URLSession.AuthChallengeDispositio
 public enum URLSessionClientError: Error {
     case noResponse
     case requestTimeout
+    case missingURL
```

Comment:
```suggestion
    case missingURLInMiddlewareRequest
```

Let's make the error a bit more verbose. I bet if people saw this error, they would be confused because they themselves had provided a url; so I think this rewording will help them understand what is really occurring.